test/test-debuglet.ts

```diff
@@ -314,51 +325,54 @@ describe('Debuglet', function() {
+          Debuglet.normalizeConfig_(config as DebugAgentConfig);
       // TODO: Debuglet.normalizeConfig_() expects 1 parameter but the original
       //       test code had zero arguments here.  Determine which is correct.
-      const compareConfig = Debuglet.normalizeConfig_(null as any as DebugAgentConfig);
+      const compareConfig =
+          Debuglet.normalizeConfig_(null as any as DebugAgentConfig);
       // The actual config should be exactly defaultConfig with only
       // maxExpandFrames adjusted.
       compareConfig.capture.maxExpandFrames = testValue;
       assert.deepEqual(mergedConfig, compareConfig);
     });
 
     it('should elaborate on inspector warning on 32 bit but not on 64 bit',
-        function(done) {
-      const projectId = '11020304f2934-a';
-      const debug = new Debug(
-          {projectId: projectId, credentials: fakeCredentials}, packageInfo);
-      const debuglet = new Debuglet(debug, defaultConfig);
-      let logText = '';
-      debuglet.logger_.info = function(s: string) {
-        logText += s;
-      };
-      nocks.projectId('project-via-metadata');
-      const scope = nock(API)
-                      .post(REGISTER_PATH)
-                      .reply(200, {debuggee: {id: DEBUGGEE_ID}});
+       function(done) {
+         const projectId = '11020304f2934-a';
+         const debug = new Debug(
+             {projectId: projectId, credentials: fakeCredentials}, packageInfo);
+         const debuglet = new Debuglet(debug, defaultConfig);
+         let logText = '';
+         debuglet.logger.info = function(s: string) {
+           logText += s;
+         };
+         nocks.projectId('project-via-metadata');
+         const scope = nock(API).post(REGISTER_PATH).reply(200, {
+           debuggee: {id: DEBUGGEE_ID}
+         });
 
-      debuglet.once('registered', function(id: string) {
-        assert.equal(id, DEBUGGEE_ID);
-        // TODO: Handle the case where debuglet.debuggee is undefined
-        assert.equal((debuglet.debuggee_ as Debuggee).project, projectId);
-        const arch = process.arch;
-        if (semver.satisfies(process.version, '>=8.5') &&
-            (arch === 'ia32' || arch === 'x86') &&
-            process.env.GCLOUD_USE_INSPECTOR) {
-          assert(logText.includes(utils.messages.ASYNC_TRACES_WARNING));
-        } else {
-          assert(!logText.includes(utils.messages.ASYNC_TRACES_WARNING));
-        }
-        debuglet.stop();
-        scope.done();
-        done();
-      });
+         debuglet.once('registered', function(id: string) {
+           assert.equal(id, DEBUGGEE_ID);
+           // TODO: Handle the case where debuglet.debuggee is undefined
+           assert.equal((debuglet.debuggee as Debuggee).project, projectId);
+           const arch = process.arch;
+           if (semver.satisfies(process.version, '>=8.5') &&
+               (arch === 'ia32' || arch === 'x86') &&
+               process.env['GCLOUD_USE_INSPECTOR']) {
```

Comment:
@gaofanmichael This change was needed to get the tests to pass.  Do you know why the tests are passing on master without this change?